src/main/java/bhs/devilbotz/subsystems/Arm.java

```diff
@@ -16,13 +17,15 @@
  * @author joshuamanoj &amp; ParkerMeyers
  */
 public class Arm extends SubsystemBase {
-  private final CANSparkMax armMotor;
+
+  /** Creates a new Arm. */
+  private CANSparkMax armMotor;
 
   /** The constructor for the arm subsystem. */
   public Arm() {
-    armMotor =
-        new CANSparkMax(
-            Constants.ArmConstants.ARM_MOTOR_CAN_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    if (RobotConfig.isCompBot()) {
```

Comment:
Is there a way we can do it other then having tens of checks for isCompBot?

Reply:
@ParkerMeyers, yes, there are lots of ways this can be done. 

1. We can keep it as-is
2. We can use `isCompBot()` just in the constructor and then check for `armMotor != null` everywhere else.
3. We could define an Arm _interface_ class and then implement the actual vs dummy one and conditionally create an instance of the correct one in `RobotContainer`.
4. Probably many others.

The current method is functional and since it isn't critical for the actual competition, we shouldn't spend too much time on it.

Reply:
I think number 3 would be the best way to implement this and to avoid the many checks

Reply:
I agree this is alot of noise to be able to conditionally create subsystems.  I prefer what is easier to read/understand for beginners.  I would choose 2 out of options above.  Or not make these changes at all?

Reply:
I vote to keep it as-is and merge to baseline.  The main goal of this PR is to minimize debug noise when using the practice bot.